pizza_cutter/des_coadd_data.py

```diff
@@ -0,0 +1,653 @@
+import os
+import shutil
+import copy
+import tempfile
+import subprocess
+
+from . import files
+
+_QUERY_COADD_TEMPLATE_BYTILE = """\
+select
+    m.tilename as tilename,
+    fai.path as path,
+    fai.filename as filename,
+    fai.compression as compression,
+    m.band as band,
+    m.pfw_attempt_id as pfw_attempt_id
+
+from
+    prod.proctag t,
+    prod.coadd m,
+    prod.file_archive_info fai
+where
+    t.tag='%(campaign)s'
+    and t.pfw_attempt_id=m.pfw_attempt_id
+    and m.tilename='%(tilename)s'
+    and m.band='%(band)s'
+    and m.filetype='coadd'
+    and fai.filename=m.filename
+    and fai.archive_name='desar2home'\n"""
+
+
+_DOWNLOAD_CMD = r"""
+    rsync \
+        -av \
+        --password-file $DES_RSYNC_PASSFILE \
+        --files-from=%(flist_file)s \
+        %(userstring)s${DESREMOTE_RSYNC}/ \
+        %(source_dir)s/
+"""
+
+_QUERY_COADD_TEMPLATE_BYTILE_DESSCI = """\
```

Comment:
@esheldon Can you look at this SQL? It appears to work, but you know the database schema's better than I do.

Reply:
It used to work I think.  It depends on which database you are using probably.  Some queries only work in desoper

Reply:
Right. It works when I use it against dessci. I just want to make sure the query is correct. Completion is a low bar.

Reply:
We can check against the file lists in existing MEDS files

Reply:
Good idea.

Reply:
Closing the loop here. I am getting this path

`OPS/finalcut/Y2A1/Y1-2363/20131010/D00242799/p01/`

from my DB queries on `desoper` but the existing meds file has

`OPS/finalcut/Y2A1/Y1-2363/20131010/D00242799/p06/`

IDK what the difference is. Any ideas @esheldon?

Reply:
Ahhhhh. This is Y3A1_COADD versus Y3A2_COADD. My tests use Y3A1_COADD but I downloaded the Y3A2_COADD. Comparing to the right MEDS file has the right paths. Merging.